custom_components/robonect/client/client.py

```
@@ -76,13 +76,32 @@ async def client_start(self):
                 self.client.auth = self.auth
 
     async def client_close(self):
-        """Properly close and cleanup the httpx client."""
+        """
+        Cleans up the HTTP client instance by setting it to None.
+        
+        This method does not explicitly close the client to avoid shutting down the shared Home Assistant HTTPX client.
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)